### PR TITLE
follow Jim's way, add route to Home, About, Landing with redux-first-…

### PR DESCRIPTION
**Pull request by fruit13ok**
…router-link
on Home, About, Landing components, added redux-first-router-link.
I use NavLink instead of Link to use some css base on https://www.npmjs.com/package/redux-first-router-link

### CODE REVIEW

**Comment by hathix** [comment]
Just inspected the behavior. It looks like it works fine. Now to look over the code.

File: frontend/src/components/About.jsx [hunk 1]
```diff
@@ -1,12 +1,29 @@
 import React, { Component, Fragment } from 'react';
-import Link from 'redux-first-router-link';
+import { Link, NavLink } from 'redux-first-router-link';
 
 class About extends Component {
   render() {
     return (
       <Fragment>
+        <button><NavLink 
+        to={{ type: 'HOME' }}
```
Comment (hathix) [inline frontend/src/components/About.jsx]: The indenting here is weird, can you make sure it follows usual indenting rules? (like have the contents of a tag be indented)

Like:

```
<Element attr=123 abc=234>
   Hi
</Element>
```

File: frontend/src/components/Home.jsx [hunk 2]
```diff
@@ -23,26 +24,46 @@ class Home extends Component {
...
+        >Home</NavLink></button>
+        <button><NavLink 
+        to={{ type: 'ABOUT' }}
+        activeStyle={{ fontWeight: "bold", color: 'purple' }}
+        exact={true}
+        strict={true}
+        >About</NavLink></button>
+        <button><NavLink 
+        to={{ type: 'LANDING' }}
+        activeStyle={{ fontWeight: "bold", color: 'purple' }}
```
Comment (hathix) [inline frontend/src/components/Home.jsx]: Again indentation

File: frontend/src/components/Home.jsx [hunk 3]
```diff
@@ -23,26 +24,46 @@ class Home extends Component {
...
+          font-family: 'Roboto', sans-serif;
+          `
+        }
+        >
+          <Intro />
+          <ControlPanel routes={routes}
+            fetchRouteConfig={this.props.fetchRouteConfig}
+            resetGraphData={this.props.resetGraphData}
+            fetchGraphData={this.props.fetchGraphData} />
+          <div className="center metricsWidth">
```
Comment (hathix) [inline frontend/src/components/Home.jsx]: why is this div empty?

File: frontend/src/components/Landing.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        <button><NavLink 
+        to={{ type: 'HOME' }}
+        activeStyle={{ fontWeight: "bold", color: 'purple' }}
+        exact={true}
+        strict={true}
+        >Home</NavLink></button>
+        <button><NavLink 
+        to={{ type: 'ABOUT' }}
+        activeStyle={{ fontWeight: "bold", color: 'purple' }}
+        exact={true}
```
Comment (hathix) [inline frontend/src/components/Landing.jsx]: again indentation

File: frontend/src/components/Home.jsx (hunk 3 above)
Comment (fruit13ok) [reply frontend/src/components/Home.jsx]: I change the empty div to Fragment. They are for React require single element, so it take a div to wrap it. Only different with Fragment is DOM does not treat Fragment as a node.

**Comment by hathix** [comment]
You just need to resolve the merge conflicts. Get rid of the yarn.lock file and fix the differences in the package-lock.json file.

**Pull request merged**